tests/test_subdomains.py

```diff
@@ -622,3 +622,32 @@ class Dummy(SubDomainSet):
 
         assert_structure(op, ['t,n', 't,n,xi_n0_blk0,yi_n0_blk0,xi_n,yi_n,zi_n'],
                          't,n,xi_n0_blk0,yi_n0_blk0,xi_n,yi_n,zi_n')
+
+    def test_sequential_implicit(self):
+        """
+        Make sure implicit dimensions define sequential iteration spaces. This is for
+        performance and potentially for correctness too (e.g., canonical openmp loops
+        forbid subiterators, which could potentially be required by a MultiSubDomain).
+        """
+
+        class Dummy(SubDomainSet):
+            name = 'dummy'
+
+        dummy = Dummy(N=0, bounds=[(), (), (), (), (), ()])
```

Comment:
Line 636 to 645 too many blank lines?

Reply:
It follows the grouping logic we always use

Reply:
the one for tests I mean